core/controllers/editor.py

```diff
@@ -775,3 +776,44 @@ def get(self, exploration_id):
         self.render_json({
             'unresolved_answers': unresolved_answers_with_frequency
         })
+
+
+class LearnerAnswerInfoHandler(EditorHandler):
+    """Handles the learner answer info for an exploration state."""
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    @acl_decorators.can_edit_exploration
+    def get(self, exploration_id):
```

Comment:
Isn't there a flag in constants.py that disables this for now? If so, use that on all the controllers to disable it for now.

Reply:
Done